src/context/WalletProvider/components/ConnectModal.tsx

```diff
@@ -37,6 +38,14 @@ export const ConnectModal: React.FC<ConnectModalProps> = ({
       </ModalHeader>
       <ModalBody>
         <Text mb={4} color='text.subtle' translation={bodyText} />
+        {error && (
+          <Alert status='info' mt={4}>
+            <AlertIcon />
+            <AlertDescription>
+              <Text translation={error} />
+            </AlertDescription>
+          </Alert>
+        )}
```

Comment:
move the error reporting above the button which makes more sense when there are `children` possibly being rendered (i.e ledger)